test/gurka/test_landmarks.cc

```diff
@@ -142,3 +148,128 @@ TEST(LandmarkTest, TestParseLandmarks) {
   EXPECT_NO_THROW({ landmarks = db.get_landmarks_by_ids({3, 2, 1}); });
   EXPECT_EQ(landmarks.size(), 3);
 }
+
+TEST(LandmarkTest, TestTileStoreLandmarks) {
+  BuildPBF();
+  // make a config, though we dont need the landmarks db in there until the next test
+  landmark_map.config =
+      test::make_config(workdir, {{"mjolnir.landmarks_db", db_path}},
+                        {{"additional_data", "mjolnir.traffic_extract", "mjolnir.tile_extract"}});
+
+  // build regular graph tiles from the pbf that we have already made, there wont be landmarks in them
+  mjolnir::build_tile_set(landmark_map.config, {pbf_filename}, mjolnir::BuildStage::kInitialize,
+                          mjolnir::BuildStage::kValidate, false);
+
+  // load one of the graphtiles via the graphtilebuilder with the deserialize option turned on
```

Comment:
some comments are good, like this one, but lots of this file has comments which were instructions for you. please clean the comments up a bit and leave only those explaining what's happening

Reply:
done :)